### ISSUE TITLE
Travis CI fails when all pep8 checks pass on my machine locally.

### ISSUE THREAD

**Issue by kofferdahl** [matthew-huber/bme590final#110]
See pull request #109 

Looking at the Travis CI logs, it looks like it's having a hard time loading conftest.py, but I'm not sure why it would be considering it works fine when I run the tests locally on my machine, and I'm sure that conftest.py has been included in the repository.

**Comment by dward2** [comment]
Did you fix this issue?  If so, out of curiosity, what did you do to fix it?  If not, let me know.

**Comment by kofferdahl** [comment]
I did fix it (sorry, I meant to reference it in the pull request), but I'm not exactly sure how I fixed it. 

Adding the blank init file like Mark suggested did not help. It seems like Travis CI was having difficulties with conftest.py after I added a fixture. It was an instance of the App class within image_processor_gui, which may not have been bale to initialize properly. I took that out and restructured my code to not need it anymore, and that seemed to help.  After restructuring my code, I found a bunch of bugs in the tests (running locally) that didn't come up before, so they may not have actually been running properly when I first tried to merge. There was also a line of code:
`matplotlib.use("QT5Agg")`
that Travis didn't like for some reason. I'm pretty sure that line of code was there from previous successful commits, but commenting that out in addition to some of the other changes I made ultimately got it to work.
